config/deploy/staging.rb

```diff
@@ -0,0 +1,23 @@
+############################################
+# Staging Server
+############################################
+
+set :stage, :staging
+set :stage_domain, "chappress-staging.chapman.edu"
+server "cprs-pre-wb01.chapman.edu", user: "wimops", roles: %w(web app db)
+set :deploy_to, "/usr/share/nginx/html"
```

Comment:
This was the default path from the nginx install (welcome screen). I believe apache defaults to  the `/var/www`. [source](https://stackoverflow.com/questions/10674867/nginx-default-public-www-location).
Shall I switch paths?